plugins/lookup/onepassword.py

```diff
@@ -52,6 +52,16 @@
           - Only works with 1Password CLI version 2 or later.
         type: str
         version_added: 7.1.0
+      connect_host:
+        description: The host for 1Password Connect. Must be used in combination with C(connect_token).
+        type: str
+        env:
+          - name: OP_CONNECT_HOST
```

Comment:
```suggestion
          - name: OP_CONNECT_HOST
        version_added: 8.1.0
```
Same for all other new options.